_rules/link-skip-block-content-7b576d.md

```diff
@@ -0,0 +1,814 @@
+		</aside>
+
+		<main id="main">
+			<h2>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h2>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+#### Passed Example 8
+
+In this [document][], the link at the start of the complementary [block of repeated content][] jumps [at its end][at the end].
+
+```html
+<html lang="en">
+	<head>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<a href="/test-assets/bypass-blocks-cf77f2/chapter2.html">Read Chapter 2</a>
+
+		<aside id="about-book">
+			<a href="#end-aside">Skip additional information</a>
+			<h1>About the book</h1>
+			<p>The Romance of the Three Kingdoms is a 14th century historical novel.</p>
+			<div id="end-aside" />
+		</aside>
+
+		<main>
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+#### Passed Example 9
+
+In this [document][], the link to skip the complementary [block of repeated content][] jumps [at its end][at the end] (at the first [perceivable content][] after it, because the `hr` element is not [palpable content][], hence not [perceivable content][]).
+
+```html
+<html lang="en">
+	<head>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<a href="/test-assets/bypass-blocks-cf77f2/chapter2.html">Read Chapter 2</a>
+
+		<aside id="about-book">
+			<a href="#main">Skip additional information</a>
+			<h1>About the book</h1>
+			<p>The Romance of the Three Kingdoms is a 14th century historical novel.</p>
+		</aside>
+
+		<main>
+			<hr />
+			<h1 id="main">Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+#### Passed Example 10
+
+In this [document][], the link to skip the complementary [block of repeated content][] jumps [at its end][at the end] (at the first [perceivable content][] after it, because the `img` element has a [semantic role][] of `presentation`, hence is not [perceivable content][]).
+
+```html
+<html lang="en">
+	<head>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<a href="/test-assets/bypass-blocks-cf77f2/chapter2.html">Read Chapter 2</a>
+
+		<aside id="about-book">
+			<a href="#main">Skip additional information</a>
+			<h1>About the book</h1>
+			<p>The Romance of the Three Kingdoms is a 14th century historical novel.</p>
+		</aside>
+
+		<main>
+			<img src="../test-assets/bypass-blocks-cf77f2/peach-garden-oath.jpg" role="presentation" alt="" />
+			<h1 id="main">Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
```

Comment:
Yes. But then the `img` would be a separate block of content and the link at the start of the `aside` would not jump to the next block of content, but to the second next.
It is not possible to consider both the `aside` and the `img` as a single block of content as a semantic segmentation requires that content inside and outside the main block of content are in separate blocks.

Reply:
I don't think so that we can consider h1 as a block of content according to the definition of block of content which state:
> ... a set of nodes from that page ....

h1 is not a set of nodes.

Reply:
A set may contain a single element 😄 (or even 0, but "block of content" requires at least one perceivable, so it cannot be the empty set).
(additionally, `<h1>Foo</h1>` actually contains 2 nodes: the `h1` element and the text node…)